perses/tests/test_samplers.py

```diff
@@ -13,6 +13,8 @@
 # IMPORTS
 ################################################################################
 
+from nose.plugins.attrib import attr
```

Comment:
I'm tagging tests we don't want to run on travis with `@attr('advanced')`.

Tests labeled as `advanced` are skipped on travis via adding the `-a '!advanced'` to `nosetests` on travis.